<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastPairwiseParametersDialog</class>
 <widget class="QDialog" name="FastPairwiseParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fast Pairwise Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Word size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QSpinBox" name="wordSizeSpinBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Use default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of best diagonals:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QSpinBox" name="bestDiagonalsSpinBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Use default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Window around best diagonals:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QSpinBox" name="windowSpinBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Use default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Gap penalty:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QSpinBox" name="gapPenaltySpinBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>Use default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Score:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="scoreByPercentageRadioButton_">
     <property name="text">
      <string>Percentage</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QRadioButton" name="scoreByAbsoluteRadioButton_">
     <property name="text">
      <string>Absolute</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>wordSizeSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>bestDiagonalsSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>windowSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapPenaltySpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FastPairwiseParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FastPairwiseParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
